assets/sass/widgets/_widget-context.scss

```diff
@@ -23,5 +23,5 @@
 }
 
 .googlesitekit-widget-context--last {
-	min-height: 80vh;
+	min-height: 100vh;
```

Comment:
This works, but is a bit unnecessary to apply on smaller screens. We can add a media query here to only apply this on larger screens as a simple fix and look at addressing this more accurately in another issue (if necessary).